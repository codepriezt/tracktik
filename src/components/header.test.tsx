import React from "react"
import {shallow} from "enzyme"
import Header from "./header"

describe('test header component ' , () => {
    it('return a string  form the document element for the toggle button ' , () => {
            expect(shallow(<Header/>)).toMatchSnapshot()
             
    })
})