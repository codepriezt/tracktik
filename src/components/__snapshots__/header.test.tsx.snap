// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test header component  return a string  form the document element for the toggle button  1`] = `<Fragment />`;
